Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM rust:1.54 as builder
+FROM rust:slim-bullseye as builder
 RUN curl -L https://yt-dl.org/downloads/2021.06.06/youtube-dl -o /usr/local/bin/youtube-dl && chmod a+rx /usr/local/bin/youtube-dl
 WORKDIR /usr/src/mrvn-bot
+RUN apt-get update && apt-get install -y libopus0 libopus-dev pkg-config ffmpeg && rm -rf /var/lib/apt/lists/*
 COPY . .
 RUN cargo install --path ./mrvn-front-discord
 
-FROM debian:buster-slim
-RUN apt-get update && apt-get install -y ca-certificates ffmpeg python && rm -rf /var/lib/apt/lists/*
+FROM bitnami/minideb:latest
+RUN apt-get update && apt-get install -y ca-certificates libopus0 libopus-dev ffmpeg && rm -rf /var/lib/apt/lists/*
```

Comment:
I haven't had the chance to try running this yet, but I noticed python isn't listed here. Is that included in the base image?

Reply:
When I was running a bot in my case everything was fine? But I will add back then if needed

Reply:
I added this python again. Also, on Debian for some reasons when you say to install just python in my case it installs the python2. So I had to explicitly specify here that user needs python3.